odl/set/pspace.py

```diff
@@ -210,7 +210,7 @@ def spaces(self):
         """A tuple containing all spaces."""
         return self._spaces
 
-    def element(self, inp=None):
+    def element(self, inp=None, cast=True):
```

Comment:
perhaps yes, I needed it for debugging but found it to be quite nice, so I kept it. Could be useful in other places as well for sure.